guidebook/sync_guidebook.py

```diff
@@ -220,6 +285,52 @@ def setup_rooms(self, rooms):
                 rid = self.rooms[room]['id']
             self.add_room(room, update, rid)
 
+    def add_x_map_region(self, map_region, update, rid, location_id):
+        if update and not self.update:
+            return
+        name = 'map-regions-%s' % rid,
+        data = {
+            'map_object': self.x_map_id,
+            'location': location_id,
+            'relative_x': map_region['x'],
+            'relative_y': map_region['y'],
+            'relative_width': map_region['w'],
+            'relative_height': map_region['h'],
+        }
+        self.add_thing('x-map-regions', name, data, update, rid)
+
+    def get_x_map_region_for_room(self, room):
```

Comment:
This method doesn't work. None of these lcations have an ID, they look like this:

```
{u'guide': 125002, u'last_updated': u'2018-03-07T02:54:50.156424+0000', u'uuid': u'0c809b8e-1f86-11e8-b6e5-0ec2a56d2ae2', u'url': u'https://builder.guidebook.com/api/locations/2725731/', u'created_at': u'2018-03-04T08:28:43.067192+0000', u'description': None, u'longitude': None, u'id': 2725731, u'address': None, u'latitude': None, u'location_type': u'gb-interactive', u'name': {u'en-US': u'Ballroom H'}}
```